controllers/postsController.js

```diff
@@ -1,12 +1,19 @@
-// import model
-Posts = require('../models/posts.json');
+// import json model (temporary)
+posts = require('../models/posts.json');
+
+//Importing the model (database)
+const postModel = require('../models/posts');
 
 // route handlers (CRUD)
 exports.viewPost = function(req, res) {
-  let post = Posts.filter(post => post.pid == req.params.pid)[0];
+  let post = posts.filter(post => post.pid == req.params.pid)[0];
```

Comment:
Oks lang ba if uppercase ang first letter to distinguish objects from model? Intentional na "Posts" and naming instead of "posts'

Reply:
Ohhh kaya pala naka uppercase yon. I'll update this.